src/content/developers/tutorials/send_token_etherjs/index.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Sending Tokens Using ethers.js
+description: Beginner friendly guide to sending tokens using ethers.js.
+author: Kim YongJun
+tags: ["ETHERS.JS", "ERC-20", "TOKENS"]
+skill: beginner
+lang: en
+sidebar: true
+published: 2021-04-06
+---
+
+
+
+## Send Token Using ethers.js(5.0) {#send-token}
+
+### In This Tutorial You'll Learn How To {#you-learn-about}
+* Import ethers.js
+* Transfer token
+* Set gas price according to the network traffic situation
+
+### To-Get-Started {#to-get-started}
+To get started, we must first import the ethers.js library into our javascript 
+1. Include ethers.js(5.0)
+
+**Installing** {#install-ethersjs}
+```shell
+/home/ricmoo> npm install --save ethers
+```
+
+ES6 in the Browser
+```html
+<script type="module">
+    import { ethers } from "https://cdn.ethers.io/lib/ethers-5.0.esm.min.js";
+    // Your code here...
+</script>
+```
+
+ES3(UMD) in the Browser
+```html
+<script src="https://cdn.ethers.io/lib/ethers-5.0.umd.min.js"
+        type="application/javascript"></script>
+```
+### Parameters {#param}
+1. **contract_address** : Token contract address(contract address is needed when the token you want to transfer is not ether)
+2. **send_token_amount** : The amount you want to send to the receiver
+3. **to_address** : The receiver's address
+4. **send_account** : The sender's address
+5. **private_key** : Private key of the sender to sign the transaction and actually transfer the tokens
+
+## Notice {#notice}
+signTransaction(tx) is removed because sendTransaction() does it internally.
```

Comment:
Similar to above, would wrap these in back-ticks to style as code:
```
`signTransaction(tx)` is removed because `sendTransaction()` does it internally.
```
`signTransaction(tx)` is removed because `sendTransaction()` does it internally.